source/config/localhost_config.json

```diff
@@ -7,5 +7,12 @@
        "server_cert": "",
        "server_key": "",
        "root_cert": ""
+    },
+    "moniker_stream_processor": {
```

Comment:
This can just be `processor_affinity`. Individual flags below call out that these are for "moniker streaming" APIs. This also gives us flexibility to add more configurations related to cpu pinning in future if needed which might not related to streaming.